includes/course-theme/class-sensei-course-theme.php

```diff
@@ -268,16 +303,29 @@ public function add_sensei_theme_body_class( $classes ) {
 	 * @access private
 	 */
 	public function enqueue_styles() {
-		Sensei()->assets->enqueue( self::THEME_NAME . '-style', 'css/sensei-course-theme.css' );
```

Comment:
One more thing I realized while exploring the widget styles issue:
In the `Sensei_Course_Theme_Editor` we're still using the old names.

Reply:
After that, I also realized an alignment issue in the editor.

<img width="1477" alt="Screen Shot 2022-02-11 at 11 23 07" src="https://user-images.githubusercontent.com/876340/153608628-e3aea903-2f49-4f3e-b006-01dec90ee938.png">

Reply:
Worked on the styling issues in [5837a3b](https://github.com/Automattic/sensei/pull/4775/commits/5837a3bad5c7442853df4bfe500a5847df8940a8)

Reply:
I still could reproduce the issue.

Always in the editor, and in the frontend when I active the "Enable theme styles" setting. Tested with Twenty Twenty-Two.

<img width="986" alt="Screen Shot 2022-02-15 at 10 26 51" src="https://user-images.githubusercontent.com/876340/154071512-0124d0aa-ef44-4b6c-8684-da606c547a8b.png">

<img width="1475" alt="Screen Shot 2022-02-15 at 10 27 08" src="https://user-images.githubusercontent.com/876340/154071538-7149aa88-fbc3-4fab-b083-153edad5e214.png">